infra/main.bicep

```diff
@@ -66,6 +68,7 @@ param openAiResourceGroupLocation string
 
 param openAiSkuName string = 'S0'
 
+@secure()
```

Comment:
The secure() should resolve some security alerts, along with the removal of the output.